src/sentry/static/sentry/app/components/events/eventTags/eventTagsPill.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {Link} from 'react-router';
+import queryString from 'query-string';
+
+import {EventTag, Meta} from 'app/types';
+import AnnotatedText from 'app/components/events/meta/annotatedText';
+import DeviceName from 'app/components/deviceName';
+import {isUrl} from 'app/utils';
+import Pill from 'app/components/pill';
+import VersionHoverCard from 'app/components/versionHoverCard';
+import InlineSvg from 'app/components/inlineSvg';
+import Version from 'app/components/version';
+
+type Props = {
+  tag: EventTag;
+  streamPath: string;
+  releasesPath: string;
+  query: any;
```

Comment:
The history package has some good types for this. You might be able to use the `Query` type.